CONTRIBUTING.md

```diff
@@ -316,13 +316,25 @@ nothing to commit, working directory clean
     add .` to add all unstaged files. Take care, though, because you can
     accidentally add files you don't want added. Review your `git status` first.
 
-6.  Commit your edits: `git commit -m "Brief Description of Commit"`. Do not add the issue number in the commit message.
+6.  Commit your edits (follow any one of the below methods):
+
+    a. Using the inbuilt script (_recommended_):
+       - We have a [tool](https://commitizen.github.io/cz-cli/) that helps you to make standard commit messages. Simply execute `npm run commit` after you have added the necessary files as mentioned in the step earlier.
+
+    b. Using Commitizen CLI:
+       - If you are already using [commitizen](http://commitizen.github.io/cz-cli/), simply doing a `git cz` works as expected too!
+
+    c. Using Git:
+       - `git commit -m "Brief Description of Commit"`.
```

Comment:
We shouldn't allow message that are not built using cz or that do not fit the our format. I suggest removing this third option